### ISSUE TITLE
Request for an Update on the recommended hardware with affiliate link to support the project

### ISSUE THREAD

**Issue by mannebk79** [blakeblackshear/frigate#15098]
I would like to buy some cameras for my surveillance system.

I checked here:
https://docs.frigate.video/frigate/hardware

to see what is recommended by the project.

Since I know the code owner is very privacy-sensitive, has a high interest in trouble-free operations, as bad hardware will add additional workload to the project, therefore I suspect the recommended products to perfectly fit most frigate users (including myself) requirements. On top of that,  this project deserves our support for making frigate available for fee.

Unfortunately the products recommended with an affiliate link are not available (anymore); instead, the jungle site (aka Amazon) recommends some reolink crap (I must know, I own 8 units from that brand, and from my own experience and in my personal opinion, they are exactly that).

For me personally, OpenIPC supported hardware would be preferable. I hate all stuff that is forcefully tied to a manufacturer; too many times I have had stuff working perfectly fine (hardware wise) but receiving no ongoing support via bug fixes and security updates and/or feature upgrades. But then, I know OpenIPC does not support a big amount of recent camera models.

Just an example, my reolink cams are around 4 years old, with exactly one update during that time in 23, that I chose to not install because the description is very informative with 3 rows with 5 words each. It's basically missing everything and all important facts, and most of the NVR stuff is still so buggy that it needs special tending by frigate and frequent reboots through PoE power cycles.

So please, due to the upcoming Black Friday, I would prefer to support the project by upgrading via an affiliate link.

My personal requirements are:

- PoE
- Ethernet

for outdoor use:
- IR with decent night vision
- weatherproof
- PTZ would be nice for some
- 2way audio to talk back

for indoor use:
- bird view units for indoor use
- small form factor to hide the unit so a burglar won't see it and therefore can't guard against. 

Thank you kindly
Cheers

**Comment by blakeblackshear** [comment]
I just checked all the links here and all of the products are available: https://docs.frigate.video/frigate/hardware/

What country are you in?

**Comment by mannebk79** [comment]
Germany

![grafik](https://github.com/user-attachments/assets/89b22b31-bea7-4de4-abe1-cb1a909c5c91)
![grafik](https://github.com/user-attachments/assets/c0a18a9d-396a-47fd-9b21-12b4c5265819)
![grafik](https://github.com/user-attachments/assets/0cdb0522-b0c8-4364-a8b3-68241ede20ac)

**Comment by mannebk79** [comment]
thanks, i have no problem sourcing from AliExpress.

what is the reason why you went with

 Dahua oder Hikvision?

what type precisely? 

and yes please keep us posted.

**Comment by Se7enair** [comment]
Main reason is that it is mentioned in the frigate docs. Not this Special Type, but the manufacturer in general. 
Also the rest of the informations I got recommended these two. 
And also made good experience with dahua at work. 

I own two Reolink, the newer one runs quite well. The older one never was stable as the New one. My brother also has a reolink which runs quite well. 
But I want to test another manufacturer. Maybe they solve problems I never relate to reolink. 
And of course, only a few euros difference made it a easy decision. 

I ordered a Dahua IPC-HFW2441S-S. Should arrive in the next few  days.
